db/index.js

```diff
@@ -1,14 +1,11 @@
 const db = require("./db");
 const { Product, Category, LineItem, Cart } = require("./models");
-const seed = require("./seed");
 
-const dbSyncAndSeed = () => {
+const dbSync = dropTables => {
   return db
     .authenticate()
```

Comment:
@5t4mp5 , could you briefly explain what does the authenticate do here, or why you add this command? Thanks

Reply:
Eliot instructed us early on to start with authenticate, to confirm we are connected to the db server before taking any other actions.